src/api/public/apidocs-new/paths/request_request_diff.yaml

```diff
@@ -0,0 +1,63 @@
+post:
+  summary: Get the diff for all by the request affected packages
```

Comment:
I don't understand this sentence. Is this `Get the diff for all packages affected in the request`? I'm not sure.

Reply:
You are right, the sentence is a bit off, will change it

Reply:
I applied your suggestion :)